inductiva/resources/machines_base.py

```diff
@@ -146,9 +146,10 @@ def start(self, **kwargs):
         try:
             self._api.start_vm_group(body=request_body)
         except inductiva.client.ApiException as e:
-            logs.log_and_exit(logging.getLogger(), logging.ERROR,
-                              "Starting machine group failed" 
-                              " with exception %s", e)
+            logs.log_and_exit(
+                logging.getLogger(), logging.ERROR,
```

Comment:
Same here. Was yapf failing on these files?

Reply:
Yapfing... I am not sure what is going on to be honest...